coreservices/partsrelationshipservice/broker-proxy-integration-tests/src/test/java/net/catenax/brokerProxy/UpdatePartRelationshipTest.java

```diff
@@ -87,18 +90,22 @@ public void updatedPartsAttributesWithNoEffectTime_failure() throws JsonProcessi
 
         assertThatJson(response)
                 .when(IGNORING_ARRAY_ORDER)
-                .isEqualTo(brokerProxyMother.invalidArgument(List.of("relationships[0].effectTime:must not be null")));
+                .isEqualTo(generateResponse.invalidArgument(List.of("relationships[0].effectTime:must not be null")));
+    }
+
+    private PartRelationshipUpdateRequest getPartRelationshipUpdateRequest(Function<PartRelationshipUpdate.PartRelationshipUpdateBuilder, PartRelationshipUpdate.PartRelationshipUpdateBuilder> f) {
+        var updateRequest = generate.partRelationshipUpdateList();
+        List<PartRelationshipUpdate> relationships = updateRequest.getRelationships().stream().map(r -> f.apply(r.toBuilder()).build()).collect(Collectors.toList());
+        return updateRequest.toBuilder().withRelationships(relationships).build();
     }
 
     @Test
     public void updatedPartsAttributesWithNoStage_failure() throws JsonProcessingException {
```

Comment:
As no checked exception thrown
```suggestion
    public void updatedPartsAttributesWithNoStage_failure() {
```